docs/data-sources/config_map_v1.md

```diff
@@ -45,6 +45,7 @@ Read-Only:
 
 
 
+
```

Comment:
tests are passing but there's some clean-up needed since this PR has some changes that are unrelated to the new `ipMode` field. Might have to do a `rebase -i` to remove the changes made by https://github.com/hashicorp/terraform-provider-kubernetes/pull/2557/commits/4ffbdec270d6485617a0bd718772ba698bd9ccfa